pages/account/cards.tsx

```diff
@@ -157,8 +167,86 @@ function CardsDisplay() {
             const theData = appContext.data.getData();
             setCardsData(theData.get("cards"));
             cardSets.forEach(cardSet => { cardSet.cards = (cards) ? cards.filter(card => card.data.category == cardSet.cardSetName) : [] });
+
+            if (cards) {
+                const filterOptions = cards.filter(card => card.displayName != "New Monster?").map(card => {
```

Comment:
I'd really like to see a more "condense" list of filters, this isn't very helpful right now I think.

Let me know what you think might work

Reply:
This is coming from a Discord feature request: https://discord.com/channels/912156088873418792/912646780372525066/1215347262842994730

I think the best would be to discuss directly with the person who asked for that feature to get to something that they'd like and that you'd be comfortable having in the code